### DISCUSSION TITLE
support webpack bundle analyzer user options

### DISCUSSION THREAD

**Discussion by cdaringe** [vercel/next.js#25464] [Ideas]
# Problem

I want to analyze my bundle using standard webpack debug provisions, like `stats.json`. However, `@next/bundle-analyzer` does not allow me to configure the stats plugin, and instead just uses hard coded, sensible defaults.

# Discussion

Allow for users to configure the BundleAnalyzerPlugin. The implementation for this is a non-breaking change, and only adds dexterity for users. Further, there are no runtime concerns with this, making it a less sensitive feature.  I opened a _draft_ PR [here #25463](https://github.com/vercel/next.js/pull/25463) to demonstrate the simplicity of adding such support.

**Comment by timneutkens** [comment]
We intentionally did not allow configuring the webpack plugin, if you want to use custom options you can add the webpack plugin directly. More info here https://github.com/vercel/next.js/pull/11066#issuecomment-599030034

**Reply by cdaringe** [reply]
Thanks @timneutkens. I searched, but somehow missed those passed issues. Appreciate the ref/link

**Reply by riccardogiorato** [reply]
Hey @timneutkens even if the idea wasn't to enable "configuring" the webpack bundle analyze plugin, after 1 year has the team intention changed on this topic? Is there another RFC with the list of possible changes to develop and discuss for it?
